Automattic/wp-calypso
# Editor: remove post-editor/iframe-preview-flag

For #3676 - this branch removes the `post-editor/iframe-preview` feature flag that is `truthy` in all configs.

__To Test__
Open a draft or a post in the editor, on a non-jetpack site, and ensure the Preview button still launches the iframe preview.
